.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: 'Bug report'
+about: 'Report a bug in JaCoCo'
+title:
+labels:
+assignes:
+
+---
+
+THIS IS A BUG TRACKER ONLY. FOR QUESTIONS PLEASE CHECK FAQ OR USE FORUM:
+
+http://www.jacoco.org/jacoco/trunk/doc/faq.html
+
+https://groups.google.com/forum/?fromgroups=#!forum/jacoco
+
+Please understand that
+ISSUES WITHOUT FOLLOWING INFORMATION WILL BE CLOSED WITHOUT COMMENTS!
+
+### Steps to reproduce
+
+* JaCoCo version:
```

Comment:
Got an idea - here we can easily exclude third-party tools such as SonarQube and Jenkins by requesting

```suggestion
* JaCoCo version: (from right bottom corner of JaCoCo report)
```

@marchof WDYT?

Reply:
👍